feature/visited/src/main/kotlin/uk/govuk/app/visited/SectionTitles.kt

```diff
@@ -2,5 +2,5 @@ package uk.govuk.app.visited
 
 data class SectionTitles(
     val today: String = "Today",
-    val thisMonth: String = "This Month",
+    val thisMonth: String = "This month",
```

Comment:
Just a thought, do we need to consider having these as localisable?

Reply:
Ideally, yeah. They really should be res strings of course, but I just could not make that work as they are created as a hash `Today = [], This month = [], October 2023` etc before hitting the screen. If you know of a way I'm all ears, but we'd couldn't find a better solution at the time 🤷